src/pkg/spinner/spinner.go

```diff
@@ -0,0 +1,16 @@
+package spinner
+
+var SpinnerChars = `-\|/`
```

Comment:
was `const`

Reply:
This is intentional, allowing the user to customize the spinning character by overriding this global variable.